projects/packages/my-jetpack/_inc/style.scss

```diff
@@ -0,0 +1,7 @@
+@import '@automattic/jetpack-base-styles/style';
+
+#my-jetpack-container {
+	.jetpack-logo {
```

Comment:
I really vote against these overrides, only if it is our only and really emergency solution. Don't think that is the case here, IMO we could wait for the `1.0` release and after that do it following the approach you were doing since it's just an `8px` difference from the current one.

Reply:
But not a big deal to move forward too, maybe highlighting that we need to move out of this as soon we can.

Reply:
Ok, that's your and a super valid opinion.
I just follow the process that we do for the development workflow organized by ourselves. [We asked for feedback](p1HpG7-ePH-p2#comment-52538), we [got feedback from designers](p1HpG7-ePH-p2#comment-52538), so thus I did this approach, which btw is the second one (closed #23074) to simply tweak the logo height to 40px.

Reply:
Yes! Agree with you @retrofox ... My only point is that we had a few discussions to avoid overriding styles down to the tree:

p9dueE-3Tq-p2
p1HpG7-dPt-p2

Mainly because of long-term technical problems it could get to us, but I understand that is a case that we're avoiding to change other packages, and need to do that here.

I'll help to provide a follow-up PR after we move with Jetpack 10.7.

So let's move forward with this one :)

Reply:
Thanks for your time and explanation and review. 🙇‍♂️